pkg/fsutils/idtools_unix.go

```diff
@@ -1,6 +1,6 @@
 // +build !windows
 
-package idtools
+package fsutils
```

Comment:
This file needs to be split up.

`LookupX()` would go to `pkg/system/user/usergroupadd_linux.go`

The file stiff can stay in fsutil